shared/chat/conversation/messages/message-popup/text/container.tsx

```diff
@@ -41,6 +41,7 @@ export default Container.connect(
     if (_canPinMessage && meta.teamname) {
       _canPinMessage = yourOperations.pinMessage
     }
+    const _canMarkAsUnread = true // always true?
```

Comment:
i think we don't need this

Reply:
hmm, when i removed this clicking the `...` would always trigger this action automatically. leaving for now..